testdata/p4_16_samples/action-register.p4

```diff
@@ -0,0 +1,26 @@
+#include <core.p4>
+
+extern Local<D> {
+  Local();
+  D read();
+  void write(in D data);
+}
+
+control C() {
+  action RW() {
+    Local<bit<32>>() local;
```

Comment:
How many instantiations does this create? I assume from the following comment on the merge request:

> This shows an example that would allow the effect of Direct registers in P4.

... that there is a separate instance per P4 table entry that invokes this action, and also per each apply block call-site that directly invokes this action, rather than there being e.g. one instance or one per P4 table or anything like that. And further, there's no kind of connection between any of the instances.

Reply:
We probably would restrict such actions to being called from tables only. At least that is the use case that is requested.
Your interpretation is the intended one otherwise.